app/views/application/post.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="card my-4">
+    <div class="card-header">
+        <a href="/users/<%=user.id%>/posts/<%=post.id%>"><h4># <%= post.Title%></h4></a>
+    </div>
+    <div class="card-body"><%=post.text%></div>
+    <div class="media-info">
+        <ul>
+            <li><a href="#">Comments: <%=post.comments_count%></a></li>
+            <li><a href="#">Likes: <%=post.likes_count%></a></li>
```

Comment:
- [ ] Instead of `likes_count` it should be `likes_counter` and because of this, an error is being thrown and the method cannot be recognized.